cmd/headscale/cli/nodes.go

```diff
@@ -40,6 +40,13 @@ func init() {
 	}
 	nodeCmd.AddCommand(expireNodeCmd)
 
+	renameNodeCmd.Flags().Uint64P("identifier", "i", 0, "Node identifier (ID)")
```

Comment:
general thought: Is this a rename, or is it a setname or something?

Reply:
I just reused `headscale namespaces rename` command to keep things familiar. I'm open for suggestions. Right now you can `headscale nodes -i 1 rename newmachinename` to set the `Nickname` field or `headscale nodes -i 1 rename` to clear it.

Reply:
Ye, its mostly fine, I didnt have a _really_ good name